src/sites/sites-react/doc/docs/react/official-theme-react.en-US.md

```
@@ -9,7 +9,7 @@ NutUI provides multiple sets of official `UI` themes by default, while allowing
 | theme description | scss file name |
 | --- | --- |
 | Jingdong APP 10.0 theme (default) | `variables.scss` |
-| Jingdong JDesign Theme | `variables-jdb.scss ` |
+| Jingdong B client Theme | `variables-jdb.scss ` |
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements
null

(MD038, no-space-in-code)